go/vt/vtadmin/api.go

```diff
@@ -1220,6 +1221,38 @@ func (api *API) GetWorkflows(ctx context.Context, req *vtadminpb.GetWorkflowsReq
 	}, nil
 }
 
+// RefreshState reloads the tablet record on the specified tablet.
+func (api *API) RefreshState(ctx context.Context, req *vtadminpb.RefreshStateRequest) (*vtadminpb.RefreshStateResponse, error) {
+	span, ctx := trace.NewSpan(ctx, "API.RefreshState")
+	defer span.Finish()
+
+	tablet, err := api.getTabletForAction(ctx, span, rbac.PutAction, req.Alias, req.ClusterIds)
```

Comment:
Opted to make this a "Put" action since while it is an action on the Tablet, it is not creating anything new.